spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/integration/AckMode.java

```diff
@@ -36,5 +37,16 @@
 	 * <p>The framework nacks the {@link com.google.pubsub.v1.PubsubMessage} when an exception
 	 * occurs while processing the message.
 	 */
-	AUTO
+	AUTO,
+
+	/**
+	 * The framework acks the {@link com.google.pubsub.v1.PubsubMessage} after it is
+	 * sent to the channel and processed successfully.
+	 * <p>The framework does nothing (no ack / no nack) of the {@link com.google.pubsub.v1.PubsubMessage}
```

Comment:
I would say "the framework does not immediately nack the message when the exception occurs, and allows the eventual redelivery to take effect."